renderdoc/os/win32/win32_stringio.cpp

```diff
@@ -240,7 +240,21 @@ void CreateParentDirectory(const string &filename)
     if(wfn[i] == L'/')
       wfn[i] = L'\\';
 
-  SHCreateDirectoryExW(NULL, wfn.c_str(), NULL);
+  std::vector<wstring> directoriesToCreate;
+  while(true)
```

Comment:
In the case of `SetCaptureFileTemplate` if called by the user from RenderDoc's API this may be a relative path, so there may be no existing directories along the way. I think this loop should be `while(!wfn.empty())` to handle that case.